src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/llm_inputs.py

```diff
@@ -58,7 +58,7 @@ class LlmInputs:
 
     DEFAULT_RANDOM_SEED = 0
     DEFAULT_PROMPT_TOKENS_MEAN = 550
-    DEFAULT_PROMPT_TOKENS_STDDEV = 250
+    DEFAULT_PROMPT_TOKENS_STDDEV = 0
```

Comment:
250 was too big, but don't we want SOME variability in our inputs lengths by default?
I guess at this point we just merge what we have and document it.